qa/run_xfstests.sh

```diff
@@ -46,7 +51,7 @@ TESTS="-g auto"	# The "auto" group is supposed to be "known good"
 
 # rbd presents geometry information that causes mkfs.xfs to
 # issue a warning.  This option avoids this class of problems.
-XFS_MKFS_OPTIONS="-l su=32k"
+#XFS_MKFS_OPTIONS="-l su=32k"
```

Comment:
Ah right, I remember you mentioned that now. Yeah, replace them with a comment noting why they're not there